docs/docsite/rst/guide_migration.rst

```diff
@@ -28,4 +31,331 @@ Module ``win_iis_website``
 
 Migrated to :ref:`microsoft.iis.website <ansible_collections.microsoft.iis.website_module>`.
 
-To complete.
+community.windows.win_iis_website has been split into two modules:
+  * ansible.microsoft.iis.website.
+  * ansible.microsoft.iis.website_info.
+
+Custom site Parameters from string ($parameters) removed from the module
+win_iis_webbinding was consolidated into microsoft.iis.website
+Parameters bind_ip, bind_port, bind_hostname moved into $bindings ip, port, hostname
+$bindings now include add/set/remove functionality
+Updated from ``Ansible.ModuleUtils.Legacy`` to ``-CSharpUtil Ansible.Basic``
```

Comment:
User's don't really need to know this as it's more a feature update with background changes rather than behaviour changes.